// Como corrigir o erro abaixo?

const dois = 2 // const estava dentro da função abaixo
function somarDois(x) {
    return x + dois;
}

function dividirDois(x) {
    return x + dois
}
console.log(somarDois(4))
console.log(dividirDois(6))